workflow/glsp-client/workflow-browser-app/package.json

```
@@ -0,0 +1,32 @@
+{
+  "private": true,
+  "name": "workflow-browser-app",
+  "version": "0.8.0",
+  "dependencies": {
+    "@theia/core": "1.4.0",
+    "@theia/editor": "1.4.0",
+    "@theia/filesystem": "1.4.0",
+    "@theia/languages": "1.4.0",
+    "@theia/markers": "1.4.0",
+    "@theia/messages": "1.4.0",
+    "@theia/monaco": "1.4.0",
+    "@theia/navigator": "1.4.0",
+    "@theia/preferences": "1.4.0",
+    "@theia/process": "1.4.0",
+    "@theia/terminal": "1.4.0",
+    "@theia/workspace": "1.4.0",
+    "@eclipse-glsp-examples/workflow-theia": "0.8.0"
+  },
+  "devDependencies": {
+    "@theia/cli": "1.4.0"
+  },
+  "scripts": {
+    "prepare": "theia build --mode development",
+    "start": "theia start --WF_GLSP=5007 --root-dir=../workspace",
+    "start:debug": "theia start --WF_GLSP=5007  --root-dir=../workspace --loglevel=debug",
```

This should be named `start:external` to align with the task call (and the minimal example). 
```suggestion
    "start:external": "theia start --WF_GLSP=5007 --root-dir=../workspace --DEBUG",
```